msbuild/Xamarin.iOS.Tasks.Core/Tasks/ValidateAppBundleTaskBase.cs

```diff
@@ -222,9 +222,7 @@ void ValidateWatchApp (string path, string mainBundleIdentifier, string mainShor
 				count++;
 			}
 
-			if (count > 0)
-				Log.LogError ("The Watch App '{0}' contains more than 1 Watch Extension.", name);
-			else if (count == 0)
+			if (count == 0)
```

Comment:
I don't know how to even go about testing that, but in the interest of getting our unit tests to pass (this check in ValidateTask was added by me as part of this patch) so that we can merge it into master, I figured it was worth relaxing the Validate task for.